src/items/projectile_manager.cpp

```diff
@@ -153,3 +153,44 @@ bool ProjectileManager::projectileIsClose(const AbstractKart * const kart,
     }
     return false;
 }   // projectileIsClose
+
+// -----------------------------------------------------------------------------
+/** Returns ABCD with D, C, B and A the numbers of bowling balls, plungers, cakes
```

Comment:
This weird decimal addition is weird, wouldn't a bitmask be much simpler? Or an array like byte[4] ?

Reply:
I've a personal dislike for arrays making me avoid them when I can. The bitmask would certainly be less convoluted.